hazelcast/src/main/java/com/hazelcast/internal/monitor/impl/PerIndexStats.java

```diff
@@ -245,6 +255,10 @@ public IndexOperationStats createOperationStats() {
      */
     long getMemoryCost();
 
+    void updateMemoryCost(long delta);
+
+    void resetMemoryCost();
```

Comment:
There is already `onClear` method with the similar semantics.

Reply:
Addressed in https://github.com/hazelcast/hazelcast/pull/25208/commits/0ed44f60ad7a43fd7e764fb117b85e9b03960459